src/Compilers/CSharp/Portable/Symbols/Source/SourceEventDefinitionAccessorSymbol.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols;
+
+/// <summary>
+/// Accessor of a <see cref="SourceFieldLikeEventSymbol"/> which is a partial definition.
+/// </summary>
+internal sealed class SourceEventDefinitionAccessorSymbol : SourceEventAccessorSymbol
```

Comment:
>SourceEventDefinitionAccessorSymbol [](http://example.com/codeflow?start=22&length=35)

Would it make sense to make this a private nested type within `SourceFieldLikeEventSymbol`?